draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1892,7 +1892,7 @@ IANA is asked to establish a registry of signature algorithm values, named
 |-------------+--------------------------------------------------------+------------------------------------------|
 | Value       | Signature Algorithm                                    | Reference / Assignment Policy            |
 |-------------+--------------------------------------------------------+------------------------------------------|
-| 0x00        | Deterministic ECDSA (NIST P-256) with HMAC-SHA256      | [RFC6979]                                |
+| 0x00        | ECDSA (NIST P-256) with HMAC-SHA256                    | [RFC6979]                                |
```

Comment:
Done.
HMAC-SHA256 seems enough, by looking at the Building Blocks section of RFC6979 (3.2).